x/gov/keeper/subspace.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/pokt-network/pocket-core/x/gov/types"
 )
 
+const maxValidatorChangeAllowedHeight = 40000
```

Comment:
The changelog reference height `28153` and the release notes refer to `45353`. Is this just kind of an arbitrary number in between?

Reply:
Arbitrary indeed, just a height that works works for testnet and mainnet.